appveyor.yml

```diff
@@ -1,6 +1,9 @@
 clone_depth: 10
 version: '{branch}-{build}'
 
+# Disable Appveyor's test discovery which is not used
+test: off
```

Comment:
Just out of curiosity, what does this do?

Reply:
Appveyor has a built-in discovery for some unit testing frameworks.
Hopefully saves a bit of time by disabling it since **ctest** is not one of them.